docs/etl/README.md

```diff
@@ -621,13 +621,14 @@ etl = (
     .extract_from(OfficeEmployeeExtractor(dataset_key="df_employee"))
     .extract_from(OfficeBirthdaysExtractor(dataset_key="df_birthdays"))
     .transform_with(
-        IntegerTransformerNC(
+        IntegerTransformer(
             dataset_input_keys="df_employee",
```

Comment:
should this be ["df_employee"] ?

Reply:
I see now that you actually allow both ways of specifying the input. I am a fan of the zen of python: `There should be one-- and preferably only one --obvious way to do it.` (https://peps.python.org/pep-0020/) but I will not insist on this. If you like it like this, then just resolve the conversation.

Reply:
I have decided to define dataset_input_keys only as List[str], and changed the rest accordingly.